parachain-template/runtime/src/xcm_config.rs

```diff
@@ -137,13 +137,14 @@ impl ShouldExecute for DenyReserveTransferToRelayChain {
 			return Err(()) // Deny
 		}
 
-		// allow reserve transfers to arrive from relay chain
+		// An unexpected reserve transfer has arrived from the Relay Chain. Generally, `IsReserve`
+		// should not allow this, but we just log it here.
```

Comment:
The "but" is in contrast to actually trying to take action to block the request here at this XCM barrier, instead of letting `IsReserve` block it.

Reply:
aha, so it's sort of "instead of denying the request here, we log a warning and let `IsReserve` take care of it"? I still think it could be phrased better but not sure what to suggest.

Reply:
Yes, because this barrier check will happen before the reserve filter is checked, but that's a bit more exhaustive so it's more idiomatic to let the rejection come from there. As this is the parachain template, and some teams that build off this _will_ want to allow `ReserveAssetDeposited`, maybe it's better to rephrase and say "you got some DOT/KSM in your sovereign account, you'll have to handle it (either reject it or account for it)".